src/components/ads/AdClickThroughButton.ts

```diff
@@ -27,16 +27,16 @@ export class AdClickThroughButton extends StateReceiverMixin(LinkButton, ['playe
     constructor() {
         super({ template: template() });
 
-        this.disabled = true;
-        this.style.display = 'none';
```

Comment:
Is the problem really that we're assigning properties/attributes in the constructor? Because then we may need to review all other components too...

Do you have a way to test this? Is it enough to just put an `<AdClickThroughButton>` inside any React app to trigger the bug?

Reply:
Yes the issue is that the attributes are assigned at the wrong time in the react workflow.
I've created a basic sample app with this component to reproduce.